src/modules/logger/logger.cpp

```diff
@@ -785,11 +785,34 @@ void Logger::run()
 				_last_sync_time = loop_time;
 			}
 
-			// update buffer statistics
-			for (int i = 0; i < (int)LogType::Count; ++i) {
-				if (!_statistics[i].dropout_start && (_writer.get_buffer_fill_count_file((LogType)i) > _statistics[i].high_water)) {
-					_statistics[i].high_water = _writer.get_buffer_fill_count_file((LogType)i);
```

Comment:
Why did you remove this? This is now broken.

Please don't just merge PR's w/o review.

Reply:
Ouch, yes it's clearly bogus, I have it fixed in the wip followup, but I can bring it to master sooner if needed. I apparently screwed up the rebase.

I would much prefer everything goes into master with a real review, but at the moment quite a lot goes ignored. I'd be up for establishing a more thorough system, but we'd need to do something like figure out code owners (and backups), commit to reasonable response times, etc. I also don't want to encourage more "rubber stamp" code reviews where we have the illusion of good process with the overhead and bureaucracy, but hardly any of the actual benefit.